src/kernel/src/arch/aarch64/interrupt.rs

```diff
@@ -130,14 +130,42 @@ pub fn init_idt() {
     todo!()
 }
 
+bitflags::bitflags! {
+    /// Interrupt mask bits for the DAIF register which changes PSTATE.
+    pub struct DAIFMaskBits: u8 {
+        /// Watchpoint, Breakpoint, and Software Step exceptions
+        const D = 1 << 3;
```

Comment:
Single letter?  Can't these at least be a short word?

Reply:
I did it that way to match the names in the architecture, but I don't mind changing it.